### PR TITLE
Added new partner - Metabit Trading

### LINKED ISSUE #4089: Adding a new adopter logo to layer5 homepage
### Current Behavior
Recently there was a new logo added to the adopters list on [meshery.io](http://meshery.io/) and we need the same logo to be added to the [layer5.io](http://layer5.io/) homepage!!

### Desired Behavior
Metabit Trading logo needs to be in the list of all the adapters of meshery on the layer5 homepage

### Screenshots / Mockups
Screenshot 2023-04-13 195700

Similar logo (highlighted one) needs to be added on layer5 homepage, here
image

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by arunkarthicknm**
**Description**

This PR fixes #4089

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit e0b8558bded4a7281eab2f647fb7e79aa81217e2 at: https://64385475a4c5610c250d5dc4--layer5.netlify.app

**Review by Udit-takkar — CHANGES_REQUESTED** [review]
@arunkarthicknm you didn't add the logo in homepage of layer5

File: src/sections/Partners/partners.js [hunk 1]
```diff
@@ -151,7 +152,18 @@ export const partners = [
     //imageRoute: "https://www.vmware.com/",
     moreInfo:
       "Layer5 enhances the Tanzu Service Mesh experience with Meshery as an application and infrastructure management platform that accelerates the adoption of open-source and cloud-native technologies by eliminating complexity with its intuitive, efficient, and flexible solution. Using Layer5's solutions along with Tanzu, engineers can self-service while enterprise IT teams offer Kubernetes as a Service to their application service teams.",
+  }, {
+    name: "Metabit Trading",
+    desc: "Metabit Trading is a technology-driven quantitative investment firm harnessing the power of artificial intelligence and machine learning.",
```
Comment (leecalcote) [inline src/sections/Partners/partners.js:157]: Thanks for rustling up a description, @arunkarthicknm 👏

Comment (arunkarthicknm) [reply src/sections/Partners/partners.js:157]: Ahh yes! 
Pardon me, will do it rn 😄 @Udit-takkar

Comment (arunkarthicknm) [reply src/sections/Partners/partners.js:157]: > Thanks for rustling up a description, @arunkarthicknm 👏

My pleasure! @leecalcote :)

**Comment by l5io** [comment]
🚀 Preview for commit 3dfa1e3528fc07bb23f37afe1d161d2991ef4310 at: https://6438e02ea4c56105ff0d5d60--layer5.netlify.app

**Comment by arunkarthicknm** [comment]
I think it's fine now  @Udit-takkar @leecalcote

**Comment by arunkarthicknm** [comment]
Is it okay? @leecalcote @Udit-takkar

**Comment by leecalcote** [comment]
Getting closer, @arunkarthicknm 👍 We'll need to ensure that the logo is presented in grayscale, so as to match the rest of the set of logos.

File: src/assets/images/partners/metabit-trading-logo .svg
Comment (leecalcote) [inline src/assets/images/partners/metabit-trading-logo .svg:1]: @arunkarthicknm these SVGs are not of a vectorized graphic, but are of a rasterized graphic.

Comment (arunkarthicknm) [reply src/assets/images/partners/metabit-trading-logo .svg:1]: Can you help me in where can I get the vectorized graphic and to grayscale it ? :)

**Comment by leecalcote** [comment]
@arunkarthicknm  The original vector format is no where to be found, so I recreated these logos in vector format.

Will you use these two versions and will you also see that the meshery.io site is updated to use these two as well?

- 
metabit-trading-gray
- 
metabit-trading-white

**Comment by arunkarthicknm** [comment]
> @arunkarthicknm The original vector format is no where to be found, so I recreated these logos in vector format.
> 
> Will you use these two versions and will you also see that the meshery.io site is updated to use these two as well?
> 
> 
> ## metabit-trading-gray
> metabit-trading-white

Sure, and on the Meshery site the style is different where first it's in grayscale after a click it goes to normal colour.

**Comment by arunkarthicknm** [comment]
The logo should be grayscale in home page, after a click it should be in normal colour. Is this the pattern ? Because for some partners like CNCF it's different from others. Please explain @leecalcote

**Comment by l5io** [comment]
🚀 Preview for commit f690a65c0778fd1d962a2f50788047ed19664ab7 at: https://643b8c5678bd4f7512f69077--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit f7dfd157f2c230be9bfedbb40fc60403b9164841 at: https://643b934ddbdcc46cf5059ff8--layer5.netlify.app

**Comment by arunkarthicknm** [comment]
@leecalcote Fine now?

**Comment by leecalcote** [comment]
@arunkarthicknm merge conflict...

**Comment by l5io** [comment]
🚀 Preview for commit 58157411dd38b66224cb13fdfc7d22f001d446fb at: https://64428cacbbf96b7132e6a3bb--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit af63f6ec15b4417b6e800c2262652080e5f19f93 at: https://6442922b7fd2fe6f34f80b16--layer5.netlify.app

**Review by leecalcote — CHANGES_REQUESTED** [review]
@arunkarthicknm delete the old SVGs. Use the new ones that I provided, please.

**Comment by arunkarthicknm** [comment]
But they're grayscale svgs shall I use them for the description too? Because there all logos are in normal colour.

**Comment by Nikhil-Ladha** [comment]
@arunkarthicknm still around to fix the issue?

**Comment by Nikhil-Ladha** [comment]
Use the black text logo for descriptions, light mode and the white text logo for dark mode.

File: src/assets/images/partners/metabit-trading-logo .svg
Comment (Nikhil-Ladha) [inline src/assets/images/partners/metabit-trading-logo .svg:1]: There is a sapce in between `trading-logo` and `.svg`, please fix that

File: src/assets/images/partners/metabit-trading-logo-gray .svg
Comment (Nikhil-Ladha) [inline src/assets/images/partners/metabit-trading-logo-gray .svg:1]: Same as above

File: src/sections/Home/Partners-home/partners-home-data.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ import rackspace from "../../../assets/images/partners/rackspace_black.svg";
 import intel from "../../../assets/images/partners/intel-grey.svg";
 import vmware from "../../../assets/images/partners/vmware.svg";
 import cncf from "../../../assets/images/partners/cncf/horizontal/black/cncf-black.svg";
+import metabittrading from  "../../../assets/images/partners/metabit-trading-logo-gray .svg";
```
Comment (Nikhil-Ladha) [inline src/sections/Home/Partners-home/partners-home-data.js:12]: Update the path here after fixing the file name

File: src/sections/Partners/partners.js [hunk 3]
```diff
@@ -3,6 +3,11 @@ import partner1 from "../../assets/images/partners/University_of_Texas_at_Austin
 import partner2 from "../../assets/images/partners/gsoc.svg";
 import partner3 from "../../assets/images/partners/UEM_Logo.jpg";
 import partner4 from "../../assets/images/partners/citrix.svg";
+
+import partner14 from "../../assets/images/partners/metabit-trading-logo .svg";
```
Comment (Nikhil-Ladha) [inline src/sections/Partners/partners.js:7]: Update the path here after fixing the file name

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]

**Comment by arunkarthicknm** [comment]
Will check it 👍🏻 @Nikhil-Ladha

File: src/sections/Home/Partners-home/partners-home-data.js (hunk 2 above)
Comment (leecalcote) [reply src/sections/Home/Partners-home/partners-home-data.js:12]: whitespace removed from filename.

**Comment by leecalcote** [comment]
After having provided SVGs to use, I'm not sure why they were converted to PNG, then wrapped in SVG, @arunkarthicknm

**Comment by l5io** [comment]
🚀 Preview for commit ad2c78bd6a17afe9904d409956d35f9377fc7e25 at: https://646d99e0929428542b9fc5c9--layer5.netlify.app

**Review by leecalcote — APPROVED** [review]

**Pull request merged**